Analysis/DataModel/include/AnalysisDataModel/HFSecondaryVertex.h

```diff
@@ -26,6 +26,14 @@ using namespace o2::analysis;
 
 namespace o2::aod
 {
+namespace hf_selcollision
+{
+DECLARE_SOA_COLUMN(WhyRejectColl, whyRejectColl, int); //!
```

Comment:
How many bits are needed here?

Reply:
I am not yet sure about that, in AliPhysics we had 16 bits (https://github.com/alisw/AliPhysics/blob/70706363d8eb5e911166640a9e83655ba7cf2d61/PWGHF/vertexingHF/AliRDHFCuts.h#L39) but I don't expect to have them all here (maybe others will be instead needed). My understanding however is that the event selection task does not have much information avaliable currently, so I only implemented the trigger class for the time being

Reply:
@fgrosa I would keep it as it is for the moment. It is difficult to foresee how much info we might want to include here.